CHANGES.md

```diff
@@ -10,6 +10,7 @@ To be released.
 
  -  Renamed `minValue`/`maxValue` parameters to `lowerBound`/`upperBound` of `IRandom.Next()` methods.   [[#555], [#558]]
  -  Renamed `IStore.IterateIndex()` method to `IterateIndexes()`.  [[#462], [#560]]
+ -  Reworked `BlockChain.GetStates()` into `GetState()` which takes only one `Address` instead of `IEnumerable<Address>` [[#510]]
```

Comment:
```suggestion
 -  Reworked `BlockChain<T>.GetStates()` into `GetState()` which takes only one `Address` instead of `IEnumerable<Address>`.  [[#510], [#563]]
```